DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,19 @@
    "metadata": {},
```

Comment:
<p>Why are you not using dask and a distributed client?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I tried it with Dask before but it runs out of memory, possibly due to bad chunking. I realised that it might be another task to find a chunking style that works in this example, and then to explain chunking here as the notebook is already long. Do you think we should nevertheless have Dask here?

Reply:
Yes. Include it. It's a good idea to ask for help though -- you don't need to figure everything yourself.

eg cc @angus-g, @dougiesquire